packages/init/src/steps/sendStarterData.ts

```diff
@@ -31,5 +34,10 @@ export async function sendStarterData(
 
   await sendCustomTypesFromStarter(repository, authTokenFromCookie, base, cwd);
 
+  if (sendDocs === false) {
```

Comment:
If the starter has documents but we don't want to send them, i'm guessing they have another use, do we want to delete them ?

From a user point of view if he/she puts the flag, I would expect my folder to be untouched i think.

Reply:
I asked come about this yesterday and we want to delete them

Reply:
Ok :)